packages/ripple-keypairs/src/index.ts

```diff
@@ -88,6 +88,9 @@ function computePublicKeyHash(publicKeyBytes: Uint8Array): Uint8Array {
 }
 
 function deriveAddressFromBytes(publicKeyBytes: Uint8Array): string {
+  if (publicKeyBytes.byteLength > 33) {
```

Comment:
Are there any disadvantages to having a stricter check?
```suggestion
  if (publicKeyBytes.byteLength != 33) {
```